### PR TITLE
Allow pagination in profiles graphql

### PR DESCRIPTION
**Pull request by akofink**

### CODE REVIEW

File: app/models/schema.rb [hunk 1]
```diff
@@ -189,6 +189,9 @@
   field :allProfiles do
     type types[ProfileType]
     description 'All profiles visible by the user'
+    argument :search, types.String, 'Search query'
+    argument :per_page, types.Int, 'Page'
+    argument :page, types.Int, 'Per page'
     resolve lambda { |_obj, _args, ctx|
       Pundit.policy_scope(ctx[:current_user], Profile).includes(:hosts)
```
Comment (dLobatog) [inline app/models/schema.rb]: This line would also need to accept the arguments to paginate

**Comment by akofink** [comment]
Updated

**Comment by akofink** [comment]
@dLobatog remind me again - what's this PR for? There must be some frontend work that needs to happen as well.

**Comment by akofink** [comment]
Closing this, as it's stale, and I don't think we need it anymore anyway.

**Pull request closed without merging**